composer.json

```diff
@@ -39,7 +39,7 @@
     ],
     "require": {
         "php": ">=5.4",
-        "doctrine/doctrine-module": "0.9.*",
+        "doctrine/doctrine-module": ">=0.8.0",
```

Comment:
Use `~0.8`. This sort of constraint will go over 1.x and 2.x (and so on), likely causing issues